File: kiki616/ABC/B/083.py
n,a,b = map(int,input().split())

# 地道君
# answer = [ i for i in range(n+1) ]
# for i in answer:

# 11絶対教のやり方(range(n+1)はif文書いて処理量を減らす方が本当はいい)
answer = [i for i in range(n+1) if a <= i and b >= i]

print(answer)

File: OCH_Tao/ABC/A/083.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
const L = input[0] + input[1];
const R = input[2] + input[3];
if (L === R) {
  console.log("Balanced");
} else if (L > R) {
  console.log("Left");
} else {
  console.log("Right");
}